fpga/mcs.wake

```diff
@@ -13,6 +13,11 @@ global def makeBitstreamPlan name vendor board topModule outputDir =
   def resources = "xilinx/vivado/2018.2", Nil
   BitstreamPlan name vendor board topModule Nil outputDir Nil resources Nil
 
+def vc707migHook =
+  def name = "vc707_1gb_mig"
+  def addResources = "vivado/vc7071gmig/1.0"
+  makeBlackBoxHook name addResources
```

Comment:
not sure if api-generator-sifive is the best place to put this. maybe this should go in fpga-shells? also I just realized that we should probably be deduping the verilog sources in the vsrcs.f file and in the environment variable. in case these things get published multiple times

Reply:
> not sure if api-generator-sifive is the best place to put this. maybe this should go in fpga-shells?

I guess it could, but to me, this api-generato-sifive/fpga is essentially the same as fpga-shells for wake files. Why would it matter which one it is in?

> I just realized that we should probably be deduping the verilog sources in the vsrcs.f file and in the environment variable. in case these things get published multiple times

Do you mean doing this in tcl?